core/src/main/java/org/mapfish/print/attribute/map/MapLayer.java

```diff
@@ -26,16 +26,26 @@
     Optional<MapLayer> tryAddLayer(MapLayer newLayer);
 
     /**
+     * Get the image buffer scaling to don't scale tiles of tiled layer.
```

Comment:
I don't understand this description. I guess you want to say `Get the image buffer scaling to not scale tiles of tiled layer.`, but I still don't understand.